syntax/parse.go

```diff
@@ -16,15 +16,21 @@ import "log"
 // Enable this flag to print the token stream and log.Fatal on the first error.
 const debug = false
 
+type Mode uint
```

Comment:
// A Mode value is a set of flags (or 0) that controls optional parser functionality.

Reply:
Done